flow/record/adapter/elastic.py

```diff
@@ -20,23 +20,40 @@
 [IP]:[PORT]: ip and port to elastic host
 [INDEX]: index to write to or read from
 [PROTOCOL]: http or https. Defaults to https when "+[PROTOCOL]" is omitted
+
+Optional arguments:
+  [INDEX]: name of the index to use
+  [VERIFY_CERTS]: verify certs of Elasticsearch instance
```

Comment:
I like the Optional arguments section, which makes it a bit more readable. We can also adopt this in other adapters.

However, now `[INDEX]` is listed twice. I made a code suggestion below, which als adds defaults:
```suggestion
[PROTOCOL]: http or https. Defaults to https when "+[PROTOCOL]" is omitted

Optional arguments:
  [INDEX]: name of the index to use (default: records)
  [VERIFY_CERTS]: verify certs of Elasticsearch instance (default: True)
```

Reply:
Good idea. Will update.